### DISCUSSION TITLE
Switch for Solar Sell on DEYE Hybrid 12kW 3P EU Inverter

### DISCUSSION THREAD

**Discussion by PerSimonsen** [kellerza/sunsynk#273] [Ideas]
Wish there was a "Solar Sell" switch in your code.
I've been trying to figure it out "watch some Youtube without success" I am a beginner 
The switch as far as I can see, address: 145
The Solar Sell switch must be used for automation that looks at the Nord Pool cost of sales when it falls below a certain  (Nord Pool -0,07 DKK) amount, and that switches off sales to the grid.
Looking forward and hope you will include it in your next update.

PS: these two are not included, the rest work fine
![Capture](https://github.com/kellerza/sunsynk/assets/50544104/4251e5e2-c892-4b9d-bda1-aee8c234df66)

**Comment by kellerza** [comment]
Have you tried adding more sensors?

https://github.com/kellerza/sunsynk/blob/main/src/sunsynk/definitions3ph.py#L191
